### DISCUSSION TITLE
vercel login issue

### DISCUSSION THREAD

**Discussion by ehizeex** [vercel/vercel#10491] [Help]
### Question

each time i try to login to my vercel account, i get this error 
 kindly help me solve this error:

We are unable to complete your signup. Please read [this guide](https://vercel.com/guides/why-can-i-not-signup) for further details.

please what do i do ?

**Comment by alvinsjoy** [comment]
I too had the same problem (#10000). You will have to contact support@vercel.com and they will get things sorted out.
